loopy/target/c/__init__.py

```diff
@@ -1264,7 +1304,24 @@ class CTarget(CFamilyTarget):
     """This target may emit code using all features of C99.
     For a target base supporting "least-common-denominator" C,
     see :class:`CFamilyTarget`.
+
+    .. attribute:: has_gnu_libc
+
+        An instance of :class:`bool` indicating whether the kernel will be
+        compiled with C-compiler that's built with GNU-libc as its standard
+        C-library.
     """
+    hash_fields = CFamilyTarget.hash_fields + ("has_gnu_libc",)
+    comparison_fields = CFamilyTarget.comparison_fields + ("has_gnu_libc",)
+
+    def __init__(self, fortran_abi=False, has_gnu_libc=False):
+        super().__init__(fortran_abi=fortran_abi)
```

Comment:
I think I'd prefer if this were a GNU-specific subclass rather than a flag.

Reply:
Yep, done!